web/public/images/weather/sun_active_error.svg

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="612px" height="612px" viewBox="0 0 612 612" style="enable-background:new 0 0 612 612;" xml:space="preserve">
+<g>
+	<g id="Sun">
+		<g>
+			<path fill="red" d="M104.914,306c0-14.478-11.75-26.229-26.229-26.229H26.229C11.75,279.771,0,291.522,0,306
```

Comment:
I think the red icon here is a bit intense and potentially distracting for people that don't want to use these layers - maybe it would be enough to just make the button disabled, so you only see the tooltip/"problem" if you actually want to enable the solar layer? :)

Reply:
Yeah,  that's a fair point 👍